shell/src/main/java/alluxio/cli/fs/command/FileSystemCommandUtils.java

```diff
@@ -43,7 +44,9 @@ private FileSystemCommandUtils() {} // prevent instantiation
   public static void setTtl(FileSystem fs, AlluxioURI path, long ttlMs,
       TtlAction ttlAction) throws AlluxioException, IOException {
     SetAttributePOptions options = SetAttributePOptions.newBuilder().setRecursive(true)
-        .setTtl(ttlMs).setTtlAction(ttlAction).build();
```

Comment:
FileSystem should do merging.

Reply:
The user is passing a specific ttl+action here we want to override options